### DISCUSSION TITLE
Encountering an error while compiling client source code

### DISCUSSION THREAD

**Discussion by Chapra78** [rustdesk/rustdesk#8417] [General]
After setup every required setup of rust environment in VS code , i am getting error -

error: couldn't read libs\portable\src\../data.bin: The system cannot find the file specified. (os error 2)

For troubleshooting i have made data.bin file under  libs\portable\src, by this error has been finished but 20  warning generated apart from this i have start debugging and got the error "Cargo run failed" 
 
Request to please provide solution.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, please search there first, many guys have asked similar question because you are not the most special though you always think you are. Usually nobody will help you with build or self-hosting here.
